index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Adaobi Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+    <nav class="container">
+      <a href="#" class="navbar-logo">Adaobi Okoye</a>
+      <a href="#" class="navbar-menu">
+        <img src="Images/Icons/Union.png" alt="Menu"/>
+      </a>

```

Comment:
- [ ] Kindly When you were making the changes, you changed the path of the image by mistake. Kindly return it to `src="Icons/Union.png"`.